styleguide/source/assets/scss/01-atoms/_links.scss

```diff
@@ -16,9 +16,13 @@ a,
   .joe__article p & {
     text-decoration: underline;
 
+    &:active,
     &:hover,
     &:focus {
-      color: $red-darker;
+      // color: $red-darker;
+      color: $white;
+      text-decoration: none;
+      background-color: $orange-darker;
```

Comment:
the background color should be $blue-darkest (#29688a)

Reply:
background-color updated to $blue-darkest